src/src/DataStructs/PluginStats.cpp

```diff
@@ -120,14 +120,16 @@ float PluginStats::getSampleExtreme(PluginStatsBuffer_t::index_t lastNrSamples,
   return res;
 }
 
-float PluginStats::getSample(PluginStatsBuffer_t::index_t lastNrSamples) const
+float PluginStats::getSample(int& lastNrSamples) const
 {
-  if ((_samples.size() == 0) || (_samples.size() < lastNrSamples)) { return _errorValue; }
+  if ((_samples.size() == 0) || (_samples.size() < abs(lastNrSamples))) { return _errorValue; }
 
   PluginStatsBuffer_t::index_t i = 0;
 
-  if (lastNrSamples < _samples.size()) {
+  if (lastNrSamples >= 0) {
     i = _samples.size() - lastNrSamples;
+  } else {
+    i = abs(lastNrSamples) - 1;
```

Comment:
Why use abs() here? Isn't that confusing to have negative indices?

Reply:
No, I'm not sure that the code will be easier with negative indices. But I'm not a programmer )